### PR TITLE
tapis-project conversion from resources into custom.

### PR DESCRIPTION
**Pull request by taoteg**
## Overview

Migrate project from Core-CMS-Resources.

## Related

…

## Changes

…

## Testing

1.

## UI

…

### CODE REVIEW

File: matcssi_cms/src/taccsite_cms/settings_custom.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # TACC: LOGOS
 ########################
 
-LOGO = [
```
Comment (taoteg) [inline matcssi_cms/src/taccsite_cms/settings_custom.py:10]: This was a random OCD edit I made.

File: matcssi_cms/src/taccsite_cms/settings_custom.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # TACC: LOGOS
 ########################
 
-LOGO = [ 
+LOGO = [
```
Comment (taoteg) [inline matcssi_cms/src/taccsite_cms/settings_custom.py:10]: This was a random OCD edit I made.

**Review by wesleyboar — COMMENTED** [review]
Skimmed. Looks good. I do wanna test before approval and merge. We can sync and remember together how to do that.

**Review by wesleyboar — APPROVED** [review]
Deployed. Success.

**Pull request merged**